src/main/java/de/retest/ui/descriptors/ConsistentRetestIdProvider.java

```diff
@@ -0,0 +1,31 @@
+package de.retest.ui.descriptors;
```

Comment:
`de.retest.ui.descriptors` is already pretty packed. What do you think of a dedicated package, e.g., `de.retest.ui.descriptors.idproviders` and move all corresponding classes in there? Since we are releasing `v5.0.0`, the breaking change would be OK.

Reply:
Fine. Can we have that in a separate PR?